src/sentry/runner/commands/devserver.py

```diff
@@ -368,7 +368,7 @@ def devserver(
 
         for topic_name, topic_data in settings.KAFKA_TOPICS.items():
             if topic_data is not None:
-                create_topics(topic_data["cluster"], [topic_name], force=True)
```

Comment:
this is there on purpose. without the devserver straight up won't start

Reply:
force isn't needed anymore, the topics are always created